src/modules/location/selectors.ts

```diff
@@ -62,7 +64,16 @@ export const getCollectionId = (state: RootState) => {
   return result ? result.params.collectionId : null
 }
 
-export const getSelectedItemId = (state: RootState) => new URLSearchParams(getSearch(state)).get('item')
+export const getSelectedItemId = (state: RootState) => {
+  const selectedItemId = new URLSearchParams(getSearch(state)).get('item')
+  if (selectedItemId) return selectedItemId
+
+  const collectionId = getSelectedCollectionId(state)
+  if (!collectionId) return null
+
+  const items = getCollectionItems(state, collectionId)
+  return isReviewing(state) ? (items.find(item => item.type === ItemType.WEARABLE) || items[0])?.id ?? null : null
```

Comment:
This could work as well for the edition flow.

wdyt about letting pre-select the first Item for all cases, not just reviewing process?